src/dump.ts

```diff
@@ -59,4 +63,44 @@ namespace Il2Cpp {
         file.close();
         ok(`dump saved to ${destination}`);
     }
+
+    export function dumpTree(path?: string, deleteIfExists: boolean = false): void {
+        const basePath = path ?? `${Il2Cpp.application.dataPath!}/${Il2Cpp.application.identifier ?? "unknown"}_${Il2Cpp.application.version ?? "unknown"}`;
+        const basePathExists = Il2Cpp.corlib.class("System.IO.Directory").method<boolean>("Exists").invoke(Il2Cpp.string(basePath));
+
+        if (!deleteIfExists && basePathExists) {
+            warn(`directory ${basePath} already exists, skipping...`);
+            return;
+        }
+
+        if (deleteIfExists && basePathExists) {
+            warn(`directory ${basePath} already exists, but tree deletion not yet supported, skipping...`);
+            return;
+        }
+
+        for (const assembly of Il2Cpp.domain.assemblies) {
+            const assemblyPath = assembly.name.replace(".", "/") + ".cs";
```

Comment:
```suggestion
            const assemblyPath = assembly.name.replaceAll(".", "/") + ".cs";
```

Reply:
Odd, yeah, thought I'd tested this. I'll need to bump `lib` to `es2021` in tsconfig.